libraries/Microsoft.Bot.Builder.Dialogs.Declarative/Resources/ResourceExplorer.cs

```diff
@@ -663,7 +665,9 @@ private void ResourceProvider_Changed(object sender, IEnumerable<Resource> resou
                     cancelReloadToken = new CancellationTokenSource();
 #pragma warning disable CA2008 // Do not create tasks without passing a TaskScheduler (this code looks problematic but excluding the rule for now, we need to analyze threading and re-entrance in more detail before trying to change it).
                     Task.Delay(1000, cancelReloadToken.Token)
+#pragma warning disable VSTHRD105 // Avoid method overloads that assume TaskScheduler.Current
```

Comment:
Can we try to address this one based on the guidance in https://github.com/Microsoft/vs-threading/blob/main/doc/analyzers/VSTHRD105.md
Leave the #pragma if you think the recommendation doesn't apply or is too risky